src/main/java/core/machines/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.machines;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started work");
```

Comment:
The message printed by the `doWork` method should be more informative. According to the checklist, it should indicate the type of machine that is working. Consider updating the message to include the specific type of `Machine`, such as 'Excavator is starting its work.'